website/client/components/settings/site.vue

```diff
@@ -365,7 +365,7 @@ export default {
       //   return network.key === networkKey;
       // });
 
-      await axios.get(`/api/v3/user/auth/social/${networkKey}`);
+      await axios.delete(`/api/v3/user/auth/social/${networkKey}`);
       // @TODO:
       // Notification.text(env.t("detachedSocial", {network: network.name}));
```

Comment:
to send a notification you can use `this.text(this.$t(stringName))` so in this case

```
 this.text(this.$t('detachedSocial', {network: network.name}));
```